python/iot_pub.py

```diff
@@ -0,0 +1,3 @@
+channel = ably.channels.get('device:rocket:1')
+channel.publish('action', { 'task': 'enable-thrusters' })
+
```

Comment:
Drop this blank line?

Reply:
https://github.com/ably/homepage-snippets/pull/15/commits/94b10c8c3cd44e43a7840dd438e983c6c7f415f3